### PR TITLE
Window popup

### PR DESCRIPTION
**Pull request by olawale-o**
This pull request contains:

- An index.html file
- A style.css file
- An assets directory with the addition of live.png and bluegit.png files
- A portfolio.js file
## The following were implemented:

- popups for both mobile and desktop screen sizes
- A button to popup project details.
- A close button to remove popup window from the view
- A javascript array that stores all the information for all projects
- Each project stored in the javascript array has a name, description, featuredImage, technologies, liveLink, and sourceLink properties.
- All projects are dynamically populated after page loads

### CODE REVIEW

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @olawale-o, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
- [ ] **The HTML Linters are missing from your repo. Please add them**

Please refer to [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to understand more about adding linters. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by olawale-o** [comment]
> Hi @olawale-o,
> 
> ### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding.
> ### To make this project even better, there are some issues that you still need to work on. You are almost there!
> ## Required Changes ♻️
> * [ ]  **The HTML Linters are missing from your repo. Please add them**
> 
> Please refer to [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to understand more about adding linters.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by olawale-o** [comment]
> Hi @olawale-o,
> 
> ### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding.
> ### To make this project even better, there are some issues that you still need to work on. You are almost there!
> ## Required Changes ♻️
> * [ ]  **The HTML Linters are missing from your repo. Please add them**
> 
> Please refer to [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to understand more about adding linters.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @smcommits
Thank you for your time and  the detailed review i would like to point out that i  used javascript linters as instructed from the previous project because no specific linters was specified for this project

**Comment by smcommits** [comment]
@olawale-o Thank you so much for reaching out. As this is a project containing HTML, CSS, and JS files. It will be ideal if you set up all the linters. Trust me, practice with linters is going to save you a lot of time as you will progress towards complex applications. If you need anything else, please feel free to leave a comment.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emirsagit** [comment]
Hi @descholar-ceo
Thank you for your time and the detailed review. I would like to point out that I used javascript for the project section HTML and CSS creating. The above section is the modal popup section and its data is populated with the data from the projects elements array via javascript. I didn't delete the default text in the model popup section because I think if the browser doesn't render javascript it will be shown. Sorry for that.

File: portfolio.js [hunk 1]
```diff
@@ -12,3 +20,163 @@ for (let i = 0; i < selectors.length; i += 1) {
...
+  return techItem;
+}
+
+for (let i = 0; i < projects.length; i += 1) {
+  const project = createProject(projects[i], i);
+  projectList.innerHTML += project;
+}
+
+const body = document.querySelector('body');
+const closeModal = document.querySelector('#closeModal');
```
Comment (GzimAsani) [inline portfolio.js:129]: - [ ] The `X` to close the modal is not appearing on front for desktop, kindly check that

Comment (GzimAsani) [reply portfolio.js:129]: - [ ] There should be some space between popup modal and the end of the page

**Review by GzimAsani — COMMENTED** [review]
### Hello  @olawale-o               :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling looks like the design :+1: 

 ## Required Changes 

- [ ] Remove the generic work section from your html from line 65 
- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**